modules/apps/frontend-data-set/frontend-data-set-api/src/main/java/com/liferay/frontend/data/set/constants/FDSTypes.java

```diff
@@ -0,0 +1,17 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.frontend.data.set.constants;
+
+/**
+ * @author Daniel Sanz
+ */
+public class FDSTypes {
```

Comment:
I'm not sure if this will be acceptable. Recently, because of https://github.com/brianchandotcom/liferay-portal/pull/157046#issuecomment-2545487145 I needed to [change](https://github.com/liferay-frontend/liferay-portal/pull/4618/commits/50204505c7f60ddb037a8a2324d208e0be01e2a1)  `FDSTimeZoneBehaviors` into `FDSTimeZoneBehaviorConstants`. This might need to be `FDSTypeConstants` or `FDSConstants`.

Reply:
sure, this naming is improvable